src/components/accordion/accordion.spec.tsx

```diff
@@ -117,4 +123,19 @@ describe('EuiAccordion', () => {
       cy.focused().contains('a link');
     });
   });
+
+  describe('Axe check', () => {
```

Comment:
Minor naming, not capitalizing AXE makes me think of actual 🪓s 
```suggestion
  describe('AXE check', () => {
```

Alternatively, if we want to be super future proof and not necessarily confuse folks who may less familiar with specific initialisms:

```suggestion
  describe('Automated accessibility check', () => {
```

Reply:
Lmao fuck, I did a bit more googling and I just realized `axe` isn't actually an initialism nor does it stand for anything, it's just literally the word `axe`, usually not capitalized??? I hate people who name JS tools so much 💀 (Plus I swear I've also seen people spell it like `aXe` in the past??)

Would strongly prefer we go with the 2nd suggestion in that case and use a generic 'automated accessibility' test name instead of a brand